spec/unit/recipes/test_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe 'jenkins_plugins::default' do
+  let(:chef_run) do
+    ChefSpec::ServerRunner.new(platform: 'centos', version: '6.6').converge(described_recipe)
+  end
+
+  it 'includes the `jenkins::java` recipe' do
+    expect(chef_run).to include_recipe('jenkins::java')
+  end
+
+  it 'includes the `jenkins::master` recipe' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.